src/Assistant.ts

```diff
@@ -310,7 +310,7 @@ function createSay(args: AllAssistantMiddlewareArgs): SayFn {
   const { channelId: channel, threadTs: thread_ts, context } = extractThreadInfo(payload);
 
   return async (message: Parameters<SayFn>[0]) => {
-    const threadContext = context.channel_id ? context : await args.getThreadContext(args);
```

Comment:
Since `args` is passed through in the [enrich function](https://github.com/slackapi/bolt-js/pull/2325/files#diff-16d5d870271a95dded1b5133e82b90335c31573c1456f8215046948dfb34588aR169), the removal of this additional pass-through shouldn't have an effect.